build.zig

```diff
@@ -31,62 +31,64 @@ pub fn build(b: *std.Build) void {
     const exclude_compressors_dfast_and_up = b.option(bool, "exclude-compressors-dfast-and-up", "") orelse false;
     const exclude_compressors_greedy_and_up = b.option(bool, "exclude-compressors-greedy-and-up", "") orelse false;
 
+    const module = b.createModule(.{
+        .target = target,
+        .optimize = optimize,
+        .strip = strip,
+        .pic = pic,
+        .link_libc = true,
+    });
     const zstd = b.addLibrary(.{
         .linkage = linkage,
         .name = "zstd",
-        .root_module = b.createModule(.{
-            .target = target,
-            .optimize = optimize,
-            .strip = strip,
-            .pic = pic,
-            .link_libc = true,
-        }),
+        .root_module = module,
```

Comment:
This makes the diff unnecessarily complicated. Can you keep the `b.createModule` call inside the `b.addLibrary` call and then just replace `zstd.addSomething` with `zstd.root_module.addSomething` wherever needed.